app/models/bridge.rb

```diff
@@ -91,7 +91,7 @@ def latest_event_data
     events.where(completed: true)
           .order(completed_at: :desc)
           .limit(1)
-          .pluck(:id, :completed_at)
+          .pluck(:id, :completed_at, :created_at)
```

Comment:
This goes for the comment above. I think we need to pluck data and in the method above, pull inbound out.